eq-publisher/src/eq_schema/Group.test.js

```diff
@@ -511,5 +511,48 @@ describe("Group", () => {
         expectedRunnerRouting
       );
     });
+
+    it("pipes in checkbox values from the pervious questions", () => {
+      const ctx = ctxGenerator(null);
+
+      Object.assign(ctx.questionnaireJson.sections[0].pages[0].answers[0], {
```

Comment:
It looks like you are defining everything on the answer. If you are not taking some its previous default values I'd prefer `ctx.questionnaireJson.sections[0].pages[0].answers[0] = ...` just so its obvious what is going on